src/towncrier/_git.py

```diff
@@ -10,18 +10,26 @@
 import click
 
 
-def remove_files(fragment_filenames: list[str], answer_yes: bool) -> None:
+def remove_files(
+    fragment_filenames: list[str], answer_yes: bool, answer_keep: bool
+) -> None:
     if not fragment_filenames:
         return
 
     if answer_yes:
         click.echo("Removing the following files:")
+    elif answer_keep:
```

Comment:
Just a very minor comment. We can merge with this.
Do we need to be verbose here ?

I guess that it can help with troubleshooting to detect which files are used for the build process... so we can keep it.

We can just return ASAP and remove_files is a NOOP for `answer_keep = Yes`

Reply:
Hmm....  This could kill two birds with one stone, not sure how would you feel about it though

```python
def remove_files(
    fragment_filenames: list[str], answer_yes: bool, answer_keep: bool
) -> None:
    if not fragment_filenames:
        return

    try:
        if answer_yes:
            click.echo("Removing the following files:")
        elif answer_keep:
            click.echo("Keeping the following files:")
            # Not proceeding with the removal of the files.
            return
        else:
            click.echo("I want to remove the following files:")
    finally:
        # Will always be printed, even for answer_keep to help with possible troubleshooting
        for filename in fragment_filenames:
            click.echo(filename)

    if answer_yes or click.confirm("Is it okay if I remove those files?", default=True):
        call(["git", "rm", "--quiet"] + fragment_filenames)
```

Reply:
Thanks for the updates.

Usually, I don't like clever code :)

But if you like it, I am ok to merge something like this... maybe with an extra comment.
But I think the code is not that hard to read.
I don't see any good reason why not to have this code approved.

Maybe remove "elif". I as general rule, I try to code with as little else/elif as possible.

```python
def remove_files(
    fragment_filenames: list[str], answer_yes: bool, answer_keep: bool
) -> None:
    if not fragment_filenames:
        return

    try:
        if answer_keep:
            # Not proceeding with the removal of the files.
            click.echo("Keeping the following files:")
            return  # See the logic from finally.

        if answer_yes:
            click.echo("Removing the following files:")
        else:
            click.echo("I want to remove the following files:")
    finally:
        # Always show the source files,
        # even for answer_keep to help with possible troubleshooting
        for filename in fragment_filenames:
            click.echo(filename)

    if answer_yes or click.confirm("Is it okay if I remove those files?", default=True):
        call(["git", "rm", "--quiet"] + fragment_filenames)
```